Assets/Scripts/Controllers/TimeManager.cs

```diff
@@ -190,4 +190,12 @@ private void InvokeEvent(Action<float> eventAction, float time)
             eventAction(time);
         }
     }
+
+    /// <summary>
+    /// Destroy this instance.
+    /// </summary>
+    public void Destroy()
```

Comment:
In the class the name seems right. And in NewWord/LoadWorld it actually needs to destroy the previous state to load the new one.